processing/data_collection/gazette/spiders/pr_curitiba.py

```diff
@@ -0,0 +1,119 @@
+from dateparser import parse
+import datetime as dt
+import re
+
+import scrapy
+
+from gazette.items import Gazette
+
+class PrCuritibaSpider(scrapy.Spider):
+    MUNICIPALITY_ID = '4106902'
+    name = 'pr_curitiba'
+    allowed_domains = ['legisladocexterno.curitiba.pr.gov.br']
+    start_urls = ['http://legisladocexterno.curitiba.pr.gov.br/DiarioConsultaExterna_Pesquisa.aspx']
+
+
+    def parse(self, response):
```

Comment:
You could put all this logic from `parse` and `scrap_year` removing the `start_urls` and implementing the `start_requests` method (https://doc.scrapy.org/en/latest/topics/spiders.html#scrapy.spiders.Spider.start_requests).